src/config/event-types.ts

```diff
@@ -1,4 +1,5 @@
 export enum EventTypes {
     PAGEVIEW = 'pageview',
+    QUALIFY = 'qualify',
     CLICK = 'click',
```

Comment:
@foxted Should this be `CUSTOM` instead of `CLICK`? Since a custom event _could_ be a click, but also could be some other non-click event.

Reply:
It's actually not even needed, since the engine itself never triggers these kind of events.